src/main/resources/avro/assaymetadata.avdl

```diff
@@ -0,0 +1,146 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines metadata objects describing experimental or analytical procedures, performed on biological objects (e.g. biosamples) or readouts from experimental procedures.
+*/
+
+protocol Assaymetadata {
+
+  import idl "common.avdl";
+  import idl "metadata.avdl";
+
+  /**
+  An analysis contains an interpretation of one or several experiments.
+  (e.g. SNVs, copy number variations, methylation status) together with
+  information about the methodology used.
+  */
+  record Analysis {
+
+    /**
+    Formats of id | name | description | accessions are described in the
+    documentation on general attributes and formats.
+    */
+    string id;
+
+    union { null, string } name = null;
+
+    union { null, string } description = null;
+
+    /**
+    The time at which this record was created.
+    Format: :ref:`ISO 8601 <metadata_date_time>`
+    */
+    union { null, string } createDateTime = null;
```

Comment:
This went through n iterations. The most verbose was ```recordCreateTime``` I think, explicitly scoping the meaning to the record, not the described object (i.e. record of a biosample, not sampling time).
To recap:
* ```created``` | ```updated``` were considered insufficient since ambiguous regarding the expected value
* same for ```recordCreated``` ..., which I had suggested before (there [had been a discussion](https://github.com/ga4gh/schemas/issues/568#issuecomment-190342645))
* not sure about the correct tense ("created" vs. "create", "updated" vs. "update")
* options
    * ```createDateTime``` | ```updateDateTime```
    * ```createdDateTime``` | ```updatedDateTime```
    * ```creationDateTime``` | ```updateDateTime```
    * ```recordCreationTime``` | ```recordUpdateTime```
    * ```timeRecordCreated``` | ```timeRecordUpdated ```
    * ```timeCreated``` | ```timeUpdated```

This should be settled ASAP, schema wide. I have no problems with ```createDateTime``` | ```updateDateTime``` myself, but also wouldn't complain about one of the other examples (or something similar).